test/generator/createTemplate.spec.js

```diff
@@ -0,0 +1,43 @@
+const createTemplate = require("../../src/generator/createTemplate");
```

Comment:
The tests files are currently named `filename.spec.js`, the convention in the project is to use `filename.test.js`